### ISSUE TITLE
Clarify criteria for accessibility-ready themes about recommending inaccessible plugins.

### ISSUE THREAD

**Issue by joedolson** [wpaccessibility/wp-a11y-docs#355]
- [x] I've checked the [existing issues for this change](https://github.com/wpaccessibility/wp-a11y-docs/issues)

**URL**
https://wpaccessibility.org/docs/accessibility-ready/theme-guidelines/no-inaccessible-plugins/

**Title**
Must not recommend or require inaccessible plugins

**What Needs Editing?**
Describe the intended changes:
1. Themes are not permitted to require plugins, so that phrasing is not relevant.
2. Need to be more clear about what "recommended" means

See [discussion in the WordPress slack](https://wordpress.slack.com/archives/CEKKABGJD/p1781924894714569)

**Comment by joedolson** [comment]
See also #352

**Comment by joedolson** [comment]
Closed in https://github.com/wpaccessibility/wp-a11y-docs/pull/356/
